profiler/profiler.go

```diff
@@ -238,6 +244,16 @@ func start(cfg Config, options ...option.ClientOption) error {
 	}
```

Comment:
```suggestion
		option.WithTelemetryDisabled(),
	}
```

Reply:
I changed to have more descriptive flag to turn off this option and replaced messy OC NeverSample setup with this option. Thank you for the suggestion!